src/components/EditOrganization/index.tsx

```diff
@@ -97,9 +118,14 @@ export default function OrganizationView({ organization, reloadOrganizationData
     if (organizationRecord.name === '') {
       setNameError('Required field.');
     } else {
+      if (!tzChangedFromDefault) {
+        organizationRecord.timeZone = defaultTimeZone;
+      }
       const response = await updateOrganization(organizationRecord);
       if (response.requestSucceeded) {
-        snackbar.toastSuccess(strings.CHANGES_SAVED);
+        snackbar.toastSuccess(
```

Comment:
Hm, I wonder whether it is too out of the ordinary though. Time zone isn't necessarily something that needs to be treated special. Will we need to have this kind of messaging for language settings as well? I may take this out depending on your thoughts here.

Reply:
When we work on the ticket of setting the org time zone, we will show a page message. So modifying the value here, it would be as modifying the user time zone in my account page, and we are not sowing anything special there

Reply:
I agree, just a generic 'saved changes' might suffice. anyways, since we will be handling the org tz initialization separately.. this is a moot point :)